sources/packages/api/src/route-controllers/institution/models/institution.user.res.dto.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  InstitutionUserAuth,
+  InstitutionUserTypeAndRole,
+  User,
+} from "src/database/entities";
+import { InstitutionLocation } from "src/database/entities/institution-location.model";
+import { UserInfo } from "../../../types";
+
+export interface InstitutionUserRespDto {
+  id?: number;
```

Comment:
Do we need the institution id to be returned here?

Reply:
This is not institution id but institution user id, this will help us for removal role/type case.